### PR TITLE
feat(statics): add token as available for switzerland

### PR DESCRIPTION
**Pull request by mukeshsp**
Ticket: EA-3157

### CODE REVIEW

**Comment by mukeshsp** [comment]
> I have my doubts about this solution, because that implies that each token that we want to be visible to Switzerland, we need to add a feature flag here, in addition to having (or not), the token in the gatekept list (bgms).
> 
> Here I think it should be transparent just as it happens with the rest of public tokens on Bitgo US and the others.

that is also added using the DEFAULT_FEATURES which common for all the tokens

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -6390,15 +6392,17 @@ export const coins = CoinMap.fromCoins([
...
   ),
   erc20(
     'e1d609f1-eb3f-4b3f-8ef6-87f434a6de83',
     'usdt',
     'Tether',
     6,
     '0xdac17f958d2ee523a2206206994597c13d831ec7',
-    UnderlyingAsset.USDT
+    UnderlyingAsset.USDT,
+    TOKEN_FEATURES_WITH_SWISS
```
Comment (grsind19) [inline modules/statics/src/coins.ts:6405]: @mukeshsp  if we add another region, this naming will become irrelevant.
How we handle this ? can you find a better way ?

Comment (mukeshsp) [reply modules/statics/src/coins.ts:6405]: In that case, we will add this feature with new variable similar to this case 
https://github.com/BitGo/BitGoJS/blob/master/modules/statics/src/coins.ts#L260

**Comment by evalenti-simtlix** [comment]
> > I have my doubts about this solution, because that implies that each token that we want to be visible to Switzerland, we need to add a feature flag here, in addition to having (or not), the token in the gatekept list (bgms).
> > Here I think it should be transparent just as it happens with the rest of public tokens on Bitgo US and the others.
> 
> that is also added using the DEFAULT_FEATURES which common for all the tokens

I mean, it is right for now, but what happens if in a near future Switzerland wants to use all ERC20 tokens? we need to add one by one the flag? I right as temporaly solution, but at least from my side, is not the best way. Also, with that, in each ticket for token onboarding need to specify if it will be used by Switzerland or not.

**Comment by mukeshsp** [comment]
> > > I have my doubts about this solution, because that implies that each token that we want to be visible to Switzerland, we need to add a feature flag here, in addition to having (or not), the token in the gatekept list (bgms).
> > > Here I think it should be transparent just as it happens with the rest of public tokens on Bitgo US and the others.
> > 
> > 
> > that is also added using the DEFAULT_FEATURES which common for all the tokens
> 
> I mean, it is right for now, but what happens if in a near future Switzerland wants to use all ERC20 tokens? we need to add one by one the flag? I right as temporaly solution, but at least from my side, is not the best way. Also, with that, in each ticket for token onboarding need to specify if it will be used by Switzerland or not.

If it is require for all the erc20 in that case we can make it default feature and that will not require addition of one by one.

**Review by aarvay — APPROVED** [review]

**Review by ravneet-bitgo — APPROVED** [review]

**Pull request merged**